tests/unit/fem/test_lin_interp.cpp

```diff
@@ -59,164 +96,1226 @@ double FdotG3(const Vector & x)
    return F * G;
 }
 
-TEST_CASE("Linear Interpolators")
+TEST_CASE("1D Identity Linear Interpolators",
+          "[IdentityInterpolator]")
 {
-   int order_h1 = 1, order_nd = 2, order_rt = 2, n = 3, dim = 3;
+   int order_h1 = 1, order_l2 = 1, n = 3, dim = 1;
    double tol = 1e-9;
 
-   Mesh mesh(n, n, n, Element::HEXAHEDRON, 1, 2.0, 3.0, 5.0);
+   FunctionCoefficient     fCoef(f1);
 
-   FunctionCoefficient       fCoef(f3);
+   for (int type = (int)Element::SEGMENT;
+        type <= (int)Element::SEGMENT; type++)
+   {
+      Mesh mesh(n, (Element::Type)type, 1, 2.0);
+
+      SECTION("Operators on H1 for element type " + std::to_string(type))
+      {
+         H1_FECollection    fec_h1(order_h1, dim);
+         FiniteElementSpace fespace_h1(&mesh, &fec_h1);
+
+         GridFunction f0(&fespace_h1);
+         f0.ProjectCoefficient(fCoef);
+
```

Comment:
Just for consistency with the rest of the tests, testing the H1(order)->H1(order+1) embedding seems appropriate to be added here

Reply:
This has been fixed as a consequence of collapsing different dimensional cases into a larger loop.